### PR TITLE
chore: update samples to give guest tokens

### PR DESCRIPTION
**Pull request by arun3528**
Going forward samples are going to provide you guest tokens when demanded 
Recording 2023-03-23 at 12 25 05

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [x] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: docs/samples/browser-plugin-meetings/app.js [hunk 1]
```diff
@@ -224,6 +227,24 @@ function unregister() {
     });
 }
 
+
+async function getGuestAccessToken() {
+
+  await axios({
+    method: 'post',
+    url: 'https://2j1g168nf8.execute-api.us-east-1.amazonaws.com/dev/guest',
```
Comment (marcin-bazyl) [inline docs/samples/browser-plugin-meetings/app.js:235]: shouldn't we be getting that url from service catalog or somewhere?

File: docs/samples/browser-plugin-meetings/app.js [hunk 2]
```diff
@@ -224,6 +227,24 @@ function unregister() {
...
+    data: {
+      name: guestName.value
+    }
+  }).then(function (response) {
+    console.log(response);
```
Comment (marcin-bazyl) [inline docs/samples/browser-plugin-meetings/app.js:240]: would be better for the log to have some meaningful message like "guest token response:"

File: docs/samples/browser-plugin-meetings/index.html [hunk 3]
```diff
@@ -48,10 +48,14 @@ <h2 class="collapsible">Meeting Management</h2>
...
                 <p class="note">
-                  NOTE: Click <a href="https://developer.webex.com/docs/getting-started" target="_blank">here</a> for access token
+                  NOTE: Click <a href="https://developer.webex.com/docs/getting-started" target="_blank">here</a> for
+                  access token
                 </p>
                 <div>
-                  <input id="access-token" name="accessToken" placeholder="Access Token" value="" type="text" required>
+                  <input id="guest-name" name="guestName" placeholder="Guest Name" value="" type="text">
+                  <button id="get-guest-token" class="btn-code" type="button"
+                    onclick=getGuestAccessToken()>generateGuestToken [prod only]</button>
```
Comment (marcin-bazyl) [inline docs/samples/browser-plugin-meetings/index.html:57]: why "generateGuestToken" and not "Generate guest token"? is generateGuestToken a name of a public SDK function?

File: docs/samples/browser-plugin-meetings/index.html [hunk 4]
```diff
@@ -48,10 +48,14 @@ <h2 class="collapsible">Meeting Management</h2>
 
               <form id="credentials">
                 <p class="note">
-                  NOTE: Click <a href="https://developer.webex.com/docs/getting-started" target="_blank">here</a> for access token
+                  NOTE: Click <a href="https://developer.webex.com/docs/getting-started" target="_blank">here</a> for
+                  access token
                 </p>
                 <div>
-                  <input id="access-token" name="accessToken" placeholder="Access Token" value="" type="text" required>
+                  <input id="guest-name" name="guestName" placeholder="Guest Name" value="" type="text">
```
Comment (marcin-bazyl) [inline docs/samples/browser-plugin-meetings/index.html:55]: is it ok if user leaves the guest name blank? should we maybe pre-populate the edit box with "guest"?

File: docs/samples/browser-plugin-meetings/app.js (hunk 2 above)
Comment (arun3528) [reply docs/samples/browser-plugin-meetings/app.js:240]: done

File: docs/samples/browser-plugin-meetings/index.html (hunk 3 above)
Comment (arun3528) [reply docs/samples/browser-plugin-meetings/index.html:57]: no its just a function in app.js so added it there e

Comment (arun3528) [reply docs/samples/browser-plugin-meetings/index.html:57]: will change it

File: docs/samples/browser-plugin-meetings/index.html (hunk 4 above)
Comment (arun3528) [reply docs/samples/browser-plugin-meetings/index.html:55]: thats a good idea to prepopulate it with some name

**Review by marcin-bazyl — APPROVED** [review]

**Pull request merged**